custom_components/owlintuition/sensor.py

```diff
@@ -288,7 +288,7 @@ def __init__(self, owldata, sensor_name, sensor_type, phase=0, zone=1, zones_cou
             self._name += f' P{phase}'
         self._zone = zone
         self._name_zone_updated = (zones_count == 1)
-        self._state = None
+        self._attr_native_value = None  # `_state` is now a cached property by HA, so we use this instead (see #36
```

Comment:
You reference `_state` but you don't use the variable anymore, does a comment that references the past still useful?

Reply:
IMHO it is, but...